packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -51,6 +63,7 @@ const headingTransformations = [
   { from: "fontWeightHeadingTitle6", to: "headingTitle6FontWeight" },
   // keep type and tailwind classes to avoid visual breaking changes
   { from: 'type="title1"', to: 'type="title0"' },
+  { from: 'type: "title1"', to: 'type: "title0"' },
```

Comment:
Not sure if I understood this specific request correctly, but it should cover [this statement](https://skypicker.slack.com/archives/C7T7QG7M5/p1724156159852009?thread_ts=1724147511.514349&cid=C7T7QG7M5):
```
And neither does <Heading as="h2" type="title2" tablet={{ type: "title1" }}>
```

Reply:
That is the same as "what if someone uses a different notation"?

This won't work either, in both cases:

```jsx
<Heading
  type={"title1"}
  tablet={{ type: `title0` }}
>...
```

I don't think we need to cover that. Double quote is the linting standard